lib/CMakeLists.txt

```diff
@@ -556,7 +556,8 @@ endif()
 if(NOT MSVC)
   target_link_libraries(volk PUBLIC m)
 endif()
-set_target_properties(volk PROPERTIES SOVERSION ${LIBVER})
+set_target_properties(volk PROPERTIES VERSION ${VERSION})
+set_target_properties(volk PROPERTIES SOVERSION ${SOVERSION})
```

Comment:
We already set `SOVERSION` but not directly. Also, it might be good to set this in `VolkVersion.cmake`.

Reply:
The PR fixes this. I mark it as resolved.